src/Azure.Functions.Cli/StaticResources/getting_started_python_function.md

```diff
@@ -0,0 +1,30 @@
+﻿## Getting Started with Azure Function
+### Last updated: March 8th 2021
+
+#### Project Structure
+The main project folder (<project_root>) can contain the following files:
+
+* **local.settings.json** - Used to store app settings and connection strings when running locally. This file doesn't get published to Azure. To learn more, see [local.settings.file](https://aka.ms/azure-functions/python/local-settings).
+* **requirements.txt** - Contains the list of Python packages the system installs when publishing to Azure.
+* **host.json** - Contains global configuration options that affect all functions in a function app. This file does get published to Azure. Not all options are supported when running locally. To learn more, see [host.json](https://aka.ms/azure-functions/python/host.json).
+* **.vscode/** - (Optional) Contains store VSCode configuration. To learn more, see [VSCode setting](https://aka.ms/azure-functions/python/vscode-getting-started).
+* **.venv/** - (Optional) Contains a Python virtual environment used by local development.
+* **Dockerfile** - (Optional) Used when publishing your project in a [custom container](https://aka.ms/azure-functions/python/custom-container).
+* **tests/** - (Optional) Contains the test cases of your function app. For more information, see [Unit Testing](https://aka.ms/azure-functions/python/unit-testing).
+* **.funcignore** - (Optional) Declares files that shouldn't get published to Azure. Usually, this file contains .vscode/ to ignore your editor setting, .venv/ to ignore local Python virtual environment, tests/ to ignore test cases, and local.settings.json to prevent local app settings being published.
+
```

Comment:
@stefanushinardi, @AnatoliB, @Hazhzeng - Please check and add more things to this.